modules/windows-workload-management.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-containers.adoc
+
+[id="windows-workload-management_{context}"]
+= Windows workload management
+
+To run Windows workloads in your cluster, you must first install the Windows Machine Config Operator (WMCO). The WMCO is a Linux-based Operator that runs on Linux-based control plane and compute nodes. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+
+.WMCO workflow
+image::wmco-lifecycle.png[WMCO workflow]
+
+Before deploying Windows workloads, you must create a Windows compute node and have it join the cluster. The Windows node hosts the Windows workloads in a cluster, and can run alongside other Linux-based compute nodes. You can create a Windows compute node by creating a Windows MachineSet to host Windows Server compute machines. You must apply a Windows-specific label to these machines that specifies a Windows OS image that has the Docker container runtime add-on enabled.
```

Comment:
> You must apply a Windows-specific label to these machines that specifies a Windows OS image that has the Docker container runtime add-on enabled.

This is done on the MachineSet. The user does not have to do anything to the Machine.